scripts/deploy-netlify.sh

```diff
@@ -18,8 +18,11 @@ echo ""
 if [ -n "$TRAVIS_TAG" ] &&
    [ -n "$TRAVIS_COMMIT" ] &&
    [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
-  # Check that the TRAVIS_COMMIT is actually on the master branch [to avoid deploying tags which are not on master]
-  git branch --contains $TRAVIS_COMMIT | grep 'master' >> /dev/null
+  # Check that the TRAVIS_COMMIT is actually on the master branch in
+  # origin to avoid deploying tags which are not on master.
+  git config --replace-all remote.origin.fetch +refs/heads/*:refs/remotes/origin/*
+  git fetch
+  git branch -r --contains $TRAVIS_COMMIT | grep 'master' >> /dev/null
```

Comment:
So I get what is happening here, but I am not sure why this would be dramatically different from what was here before in this case

Reply:
So in practice, before that we were not fetching the remote branches and only checking against the "local" master. The local master actually does not exist. Travis only has the one commit that it is building against.

Reply:
The important line is 23rd actually because it "links" again.
See this https://stackoverflow.com/questions/34405047/how-do-you-merge-into-another-branch-using-travis-with-git-commands/44036486 for details.